lib/jsonapi/active_model_error_serializer.rb

```diff
@@ -49,8 +73,10 @@ class ActiveModelErrorSerializer < ErrorSerializer
         { pointer: "/data/attributes/#{error_key}" }
       elsif rels.include?(error_key)
         { pointer: "/data/relationships/#{error_key}" }
+      elsif error_key == :base
+        { pointer: '/data' }
       else
-        { pointer: '' }
+        { pointer: nil }
```

Comment:
Previous discussion: https://github.com/stas/jsonapi.rb/pull/45#discussion_r604775408

`pointer: ''` references the document root, which strikes me as the wrong fallback behavior here?

Original commit: https://github.com/stas/jsonapi.rb/commit/43cbaeee6c6f89a5bc4dc7d0d92143f27ae242d9

Reply:
This should return an empty object, since there's no source to point to.